app/models/perseverance_scraper.rb

```diff
@@ -0,0 +1,77 @@
+class PerseveranceScraper
+  require "open-uri"
+  require 'json'
+  BASE_URL = "https://mars.jpl.nasa.gov/msl/multimedia/raw/"
```

Comment:
This is still the curiosity BASE_URL, furthermore it is not used down the line

Reply:
I changed it - looks like it is used when generating coverage/index.html.